apps/web/components/RedirectToLastSolved.tsx

```diff
@@ -10,12 +10,9 @@ export const RedirectToLastSolved = ({ trackId }: { trackId: string }) => {
   const router = useRouter();
 
   useEffect(() => {
-    const problemId = getFirstProblemForTrack(trackId);
-    try {
+    getFirstProblemForTrack(trackId).then((problemId) => {
       router.replace(`/tracks/${trackId}/${problemId}`);
-    } catch (e) {
-      console.error(e);
-    }
+    });
   }, [trackId]);
```

Comment:
@N4r35h if the trackId is coming as a prop then we can rule it out of the dependency, right? Since it's never going to change during the component lifecycle.

Reply:
this is correct, since this has been merged and this doesn't alter the outcome what so ever lets just hope someone in the future removes it